english/explore-outside/help-child.md

```diff
@@ -0,0 +1,13 @@
+As you approach the small, green child fishing in the marshmallow boat, you realize
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.